curriculum/challenges/english/07-scientific-computing-with-python/learn-interfaces-by-building-an-equation-solver/664e4a590b52ba8d2adff19f.md

```diff
@@ -0,0 +1,129 @@
+---
+id: 664e4a590b52ba8d2adff19f
+title: Step 33
+challengeType: 20
+dashedName: step-33
+---
+
+# --description--
+
+The discriminant of a quadratic equation in the form \\( ax^2 + bx + c = 0 \\), usually indicated by the capital Greek letter delta, is equal to \\( Δ = b^2 - 4ac \\).
+
+Within the `QuadraticEquation` class, define an `__init__` method. Use a `super()` call to inherit the `__init__` method defined inside the interface and define a new attribute named `delta`, which stores the value of the discriminant of the equation.
```

Comment:
Note: `super()` is introduced in the vector space, and this would determine the order of this project.

Reply:
This should come just after the vector space :+1: